services/jaeger/docker-compose.letsencrypt.yml

```diff
@@ -1,7 +1,7 @@
-version: '3.7'
+version: "3.7"
 services:
   jaeger:
     deploy:
       labels:
         - traefik.http.routers.jaeger.rule=Host(`${MONITORING_DOMAIN}`) && PathPrefix(`/jaeger`)
-        - traefik.http.routers.graylog.tls.certresolver=lehttpchallenge
+        - traefik.http.routers.jaeger.tls.certresolver=lehttpchallenge
```

Comment:
Please remind me: is jaeger operative or we need to more to higher version of python to take full advantage of it?

Reply:
we need python 3.7 to take full advantage